tests/search_aggregates_v5/test_aggregation/main_summary_v4.ndjson

```diff
@@ -0,0 +1,10 @@
+{"distribution_id": null, "locale": "de", "country": "US", "default_search_engine": "google", "submission_date_s3": "2017-01-01", "search_cohort": null, "scalar_parent_browser_search_with_ads": null, "search_counts": {"list": [{"element": {"engine": "google", "source": "urlbar", "count": 4}}]}, "client_id": "b", "app_version": "54.0.1", "active_addons": {"list": [{"element": {"addon_id": "random@mozilla.com", "version": "0.1", "name": "random"}}, {"element": {"addon_id": "followonsearch@mozilla.com", "version": "0.9.5", "name": "Follow-on Search Telemetry"}}]}, "scalar_parent_browser_search_ad_clicks": null}
```

Comment:
The nulls here are redundant (with the schema being specified via `main_summary_v4.schema.json`), the keys should just be omitted entirely.